src/unified_extraction_pipeline.py

```
@@ -139,7 +139,51 @@ def process_document(self, file_path: Path) -> ExtractedDocument:
             # PATH B: Fallback to Standard Unstructured
             # This infers hierarchy based on font sizes, etc.
             logger.info("🛡️ PATH B: Fallback to Unstructured Native Hierarchy")
-            return self.unstructured_extractor.extract(file_path)
+            return self._process_fallback_structure(file_path)
+
+    def _process_fallback_structure(
+            self,
+            file_path: Path,
+            start_time: float
```

Ruff (0.14.8)

Unused method argument: `start_time`

(ARG002)